lib/merchant_repository.rb

```diff
@@ -85,4 +85,15 @@ def average_average_price_per_merchant
     total_avg_item_prices = all.sum { |merchant| average_item_price_for_merchant(merchant.id) }
     average(total_avg_item_prices, all.length).round(2)
   end
+
+  def top_revenue_earners(num)
+    sorted = all.sort_by do |merchant|
+      -merchant.revenue
```

Comment:
Is this '-' doing something here? I don't know what it does.

Reply:
It's used to invert the order. If it wasn't there it would sort ascending, but we need it to be a descending order so it starts with the highest value first.